tasks/release.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(gulp, plugins, _, config) {
 
 	var TEMP_PATH = path.join(process.cwd(), 'temp');
 
-	var BRANCH_DEV = '1.x-develop';
+	var BRANCH_STABLE = '1.x-stable';
```

Comment:
So I probably made a mistake here. I thought we weren't updating `stable` any more as it is kind of pointless (I don't think we've ever cut a release that was from `stable` and not `master` in the last couple of years).

In my [redundant version of this PR](https://github.com/liferay/clay/pull/3967) I removed this, and over on that [other issue](https://github.com/liferay/clay/issues/3958), I actually [_deleted_ the `1.x-stable` branch](https://github.com/liferay/clay/issues/3958#issuecomment-788733603) because I thought it was old/stale (it was pointing at the last release from over two years before). If you want to resurrect it, you can do so with something like:

```
git checkout 1.x-stable # seeing as you probably still have it locally
git merge 1.x --ff-only
git push upstream-rw 1.x-stable --dry-run
git push upstream-rw 1.x-stable
```

If you don't still have `1.x-stable` in your local clone, you can recreate it easily enough:

```
git checkout 1.x
git checkout -b 1.x-stable
git push upstream-rw 1.x-stable --dry-run
git push upstream-rw 1.x-stable
```

Reply:
Removing it sounds like the better option. I'll take what you did in the redundant pr and add it here.